test/verify/check-packagekit

```diff
@@ -298,6 +297,68 @@ class TestUpdates(MachineCase):
 
         self.allow_restart_journal_messages()
 
+    def testInfoTruncation(self):
+        b = self.browser
+        m = self.machine
+
+        # update with not too many binary packages
+        for i in range(8):
+            self.createPackage("coarse{:02}".format(i), "1", "1", install=True)
+            self.createPackage("coarse{:02}".format(i), "1", "2", changes="make it greener")
+
+        # update with lots of binary packages
+        for i in range(25):
+            self.createPackage("fine{:02}".format(i), "1", "1", install=True)
+            self.createPackage("fine{:02}".format(i), "1", "2", changes="make it better")
+
+        # update with long changelog
+        changelog = ""
+        for i in range(30):
+            changelog += " - Things change #{:02}\n".format(i)
+        self.createPackage("verbose", "1", "1", install=True)
+        self.createPackage("verbose", "1", "2", changes=changelog)
+
+        self.enableRepo()
+
+        m.start_cockpit()
+        b.login_and_go("/updates")
+
+        b.wait_present("table.listing-ct")
+        b.wait_in_text("table.listing-ct", "Things change #05")
+
+        # "coarse" package list should be complete
+        t = b.text("#app .listing-ct tbody:nth-of-type(1) th")
+        self.assertIn("coarse00", t)
+        self.assertIn("coarse07", t)
+        self.assertNotIn("more", t)
+
+        # "fine" package list should be truncated
+        t = b.text("#app .listing-ct tbody:nth-of-type(2) th")
+        self.assertIn("fine00", t)
+        self.assertIn("fine10", t)
+        self.assertNotIn("fine20", t)
+        b.wait_present("#app .listing-ct tbody:nth-of-type(2) th a")
+        b.wait_in_text("#app .listing-ct tbody:nth-of-type(2) th a", "10 more")
+
+        # verbose changelog should be truncated
+        t = b.text("#app .listing-ct tbody:nth-of-type(3) td:nth-of-type(3)")
+        self.assertIn("Things change #00", t)
+        self.assertIn("Things change #04", t)
+        self.assertNotIn("Things change #07", t)
+        self.assertNotIn("Things change #20", t)
+        b.wait_present("#app .listing-ct tbody:nth-of-type(3) td:nth-of-type(3) a")
+        b.wait_in_text("#app .listing-ct tbody:nth-of-type(3) td:nth-of-type(3) a", "More information")
+
+        # expand fine package list by clicking on the "more" link
+        b.click("#app .listing-ct tbody:nth-of-type(2) th a")
+        b.wait_in_text("#app .listing-ct tbody:nth-of-type(2) th", "fine24")
+        b.wait_not_present("#app .listing-ct tbody:nth-of-type(2) th a")
```

Comment:
see above... if this should only expand the package list, you could add a check here to make sure that only the package list was expanded and not the changelog